src/sid/contacts.py

```diff
@@ -33,13 +33,20 @@ def calculate_contacts(contact_models, contact_policies, states, params, date):
         model_specific_contacts = validate_return_is_series_or_ndarray(
             model_specific_contacts, when=f"Contact model {model_name}"
         )
-        if model_name in contact_policies:
-            cp = contact_policies[model_name]
-            policy_start = pd.Timestamp(cp["start"])
-            policy_end = pd.Timestamp(cp["end"])
+        for policy in contact_policies.values():
```

Comment:
if this loop is time relevant, we could change the `contact_policies` from the user input to a dictionary with the contact model names as keys and lists of policies as values.

Reply:
This won't even appear in a snakeviz profile.